src/functionalTest/java/uk/gov/hmcts/reform/divorce/maintenance/GetCaseFromCcdTest.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.hmcts.reform.divorce.maintenance;
```

Comment:
You could put this in the uk.gov.hmcts.reform.divorce.ccdcase directory

Reply:
done